core/new-gui/src/app/workspace/service/undo-redo/undo-redo.service.ts

```diff
@@ -123,4 +130,25 @@ export class UndoRedoService {
   public clearRedoStack(): void {
     this.redoStack = [];
   }
+
+  private sendCommand(update: string): void {
+    if (this.workflowCollabService.getSendData()) {
+      this.workflowCollabService.sendCommand(update);
+    }
+  }
+
+  private handleRemoteChange(): void {
+    const self = this;
```

Comment:
can we avoid this `self = this` pattern?

Reply:
Done.